Celeste.Mod.mm/Patches/FinalBoss.cs

```diff
@@ -30,14 +35,29 @@ public patch_FinalBoss(EntityData e, Vector2 offset)
         [PatchBadelineBossOnPlayer] // ... except for manually manipulating the method via MonoModRules
         public new extern void OnPlayer(Player player);
 
+        public extern void orig_PushPlayer(Player player);
+
+        public void PushPlayer(Player player) {
```

Comment:
```suggestion
        private extern void orig_PushPlayer(Player player);

        private void PushPlayer(Player player) {
```

Reply:
Weird I just accepted that but it rolled back I'll accept that again

Reply:
:) it's me deleted them and merged two suggestions into one

Reply:
Btw there is a new Github feature that allowed me to accept both of the changes at once into the same commit